src/components/TopBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './TopBar.css';
+import logo from '../logoTopBar.png';
+
+class TopBar extends Component {
+  constructor() {
+    super();
+    // TODO: Replace these information by the one returned from the OpenID api
+    this.state = {
+      picture: 'https://avatars2.githubusercontent.com/u/1298871?s=40&v=4',
+      name: 'Thiago Coimbra Lemos',
+      username: 'tclemos',
+    };
+  }
+
+  render() {
+    const { picture, name, username } = this.state;
+
+    return (
+      <div className="top-bar">
+        <div className="logo-container">
+          <img className="logo" src={logo} alt="huskyCI" />
+        </div>
+        <div className="menu-container">
+          <ul>
+            <li>Item 1</li>
+            <li>Item 2</li>
+            <li>Item 3</li>
+            <li>Item 4</li>
+          </ul>
+        </div>
+        <div className="user-container">
+          <div className="user-info">
+            <div className="user-username">
+              {username}
+            </div>
+            <div className="user-name">
+              {name}
+            </div>
+          </div>
+          <div className="user-picture-container">
+            <img className="user-picture" src={picture} alt="user" />
```

Comment:
I believe we could improve the `alt` text or leave it as an empty string (`alt=""`), so assistive technologies, such as screen readers, can skip the image itself and don't bother the user.

Reply:
We could do something like this:

```js
<img className="user-picture" src={picture} alt=`Profile picture of ${this.state.user}` />
```

Reply:
Tried to follow your second suggestion but, as you can see, there is a suggestion message from `eslint` to avoid using words like `picture`, `image` or any other word that represents an image. Theoretically a screen-reader would read this as a `User` image, is that not ok?

![image](https://user-images.githubusercontent.com/1298871/66319627-d7661800-e8f3-11e9-9c92-8a94ca289e01.png)

Reply:
The lint is right, but the way it is right now the screen reader would read like this: `Image, user`, which is fine, but as I said I believe we could improve it. Maybe we can just remove the word `picture` and leave like: `Profile of ${name}`.

What do you think?

Reply:
That's fine, already changed to that!